incompatibility/incompatibility-report/incomp-scanning.go

```diff
@@ -43,24 +45,42 @@ func CreateIncompReport(env *plugins.Environment, reportType Report) {
 		}
 		openAPIdocument := &openapiv3.Document{}
 		err := proto.Unmarshal(model.Value, openAPIdocument)
-		if err != nil {
-			continue
-		}
-		printIncompatibilityReport(ScanIncompatibilities(openAPIdocument))
+		env.RespondAndExitIfError(err)
+		incompatibilityReport := ScanIncompatibilities(openAPIdocument)
+		writeProtobufMessage(incompatibilityReport, env)
+		env.RespondAndExit()
+	}
+	env.RespondAndExitIfError(errors.New("no supported models for incompatibility reporting"))
+}
 
+func writeProtobufMessage(incompatibilityReport *IncompatibilityReport, env *plugins.Environment) {
+	incompatibilityReportBytes, err := proto.Marshal(incompatibilityReport)
+	env.RespondAndExitIfError(err)
```

Comment:
Do we want to handle the error exit at the top level in create incompat report?

Reply:
I think if we handle this in the in createincompreport it would still require reporting back to the user that we were unable to create the message which would still require the call to env.respondandexitiferror() so I think functionally it serves the same purpose and keeps the scope of this function to handling the creation (error handling included) of the file but if there's some other value that I'm not recognizing let me know